### DISCUSSION TITLE
Welcome to halo Discussions!

### DISCUSSION THREAD

**Discussion by ruibaby** [halo-dev/halo#2164] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by guqing** [comment]
good job

**Comment by Bestwishesky** [comment]
how to use it

**Comment by MrYeungYik** [comment]
no segments* file found in LockValidatingDirectoryWrapper,how to fix it

**Reply by JohnNiang** [reply]
It would be better if you tell us what version you are using and how to reproduce it.

**Comment by Mrff520** [comment]
I want to be free, safe, and secure. Browse securely after my history which made me declare bankruptcy

**Reply by Mrff520** [reply]
Noted.
